cmd/tools/doctor/restZapiDiff.go

```diff
@@ -113,8 +113,8 @@ func metricValueDiff(metricName string) {
 	keyIndexes := make([]int, 0)
 
 	// These plugin generated metrics are node scoped.
-	environmentSensorMetrics := strings.Join([]string{"environment_sensor_power", "environment_sensor_ambient_temperature", "environment_sensor_max_temperature",
-		"environment_sensor_average_temperature", "environment_sensor_average_fan_speed", "environment_sensor_max_fan_speed", "environment_sensor_min_fan_speed"},
+	environmentSensorMetrics := strings.Join([]string{"environment_sensor_power", "environment_sensor_average_ambient_temperature", "environment_sensor_min_ambient_temperature", "environment_sensor_max_temperature",
```

Comment:
minor:we have enough of these that it probably makes sense to put each on a separate line and sort